src/app/commands/cmd_import_sprite_sheet.cpp

```diff
@@ -144,16 +152,35 @@ class ImportSpriteSheetWindow : public app::gen::ImportSpriteSheet
       std::max<int>(1, h));
   }
 
+  gfx::Size getPaddingFromEntries() {
+    int padW = horizontalPadding()->textInt();
+    int padH = verticalPadding()->textInt();
+    if (padW < 0) {
+      horizontalPadding()->setText("0");
```

Comment:
There is no need to change the text of the `horizontalPadding` field:
```suggestion
```
It's better if we avoid modifying data when a `get` member functions is called.